scripts/convert_to_geomdat.py

```diff
@@ -0,0 +1,205 @@
+Written by Alice Walker and Ruibin Liang
+
+This script converts from Amber restart format to AIMS Geometry.dat files. It assumes your restart file has velocities.
+
+To run, just give it the paths to your prmtop, restart data and desired output data as below.
+
+main('test_data/parm.prmtop','test_data/','test_data/outs/')
+
+"""
+
+import argparse
+from pathlib import Path
+
+import numpy as np
+
+
+def read_prmtop(prmpath: str):
+
+    """This function parses a prmtop to extract atomic types/names and masses.
+
+    Params:
+      prmpath: the path to the prmtop file
+
+    Returns:
+       namelist: the atom names as a vector of strings.
+       atommass: atomic masses as a vector of floats.
+
+    """
+
+    namelist = []
+    atommass = []
+    with open(prmpath) as f:
+        for line in f:
+            if "ATOM_NAME" in line:
+                line = next(f)
+                for line in f:
+                    if "CHARGE" in line:
+                        break
+                    else:
+                        line = line.strip("\n")
+                        for c in range(0, len(line), 4):
+                            namelist = namelist + [line[c : c + 3]]
+            if "MASS" in line:
+                line = next(f)
+                for line in f:
+                    if "ATOM_TYPE_INDEX" in line:
+                        break
+                    else:
+                        a = line.split()
+                        for x in a:
+                            atommass.append(float(x))
+    return namelist, atommass
+
+
+def read_rst(rstpath: str):
+
+    """This function parses a restart file to extract coordinates and velocities.
+
+    Params:
+      rstpath: the path to the restart file(s) to be processed. Can handle instances in subdirectories.
+
+    Returns:
+      atomcoord: the xyz coordinates as an N by 3 array.
+      atomvel: the velocities as an N by 3 array.
+
+    """
+
+    atomcoord = np.array([])
+    atomvel = np.array([])
+    atomnum = 0
+    with rstpath.open() as f:
+        f.readline()
+        line = f.readline()
+        atomnum = int(line.split()[0])
+        atomcoord = np.zeros((atomnum, 3))
+        atomvel = np.zeros((atomnum, 3))
+        for j in range(0, int((atomnum + 1) / 2)):
+            line = f.readline()
+            a = line.split()
+            atomcoord[j * 2] = np.array([float(a[0]), float(a[1]), float(a[2])])
+            if len(a) == 6:
+                atomcoord[j * 2 + 1] = np.array([float(a[3]), float(a[4]), float(a[5])])
+        for j in range(0, int((atomnum + 1) / 2)):
+            line = f.readline()
+            a = line.split()
+            atomvel[j * 2] = np.array([float(a[0]), float(a[1]), float(a[2])])
+            if len(a) == 6:
+                atomvel[j * 2 + 1] = np.array([float(a[3]), float(a[4]), float(a[5])])
+    return atomcoord, atomvel, atomnum
+
+
+def output_geom(
+    namelist: list,
+    atommass: list,
+    atomcoord: np.array,
+    atomvel: np.array,
```

Comment:
For these more complex types to keep ourselves python3.6+ compatible you'll need:

```
from typing import List
from numpy.typing import ArrayLike
```
And then modify your function to be:
```
def output_geom(
    namelist: List[whatever type the list contains, probably int or str for this),
    atommass: List[type list will contain]
    atomcoord: ArrayLike
    atomvel: ArrayLink
...
```